### ISSUE TITLE
Timeline names not rendering properly ?

### ISSUE THREAD

**Issue by jhw** [metrasynth/radiant-voices#63]
Hi,

Trying to give visible names to the different patterns in my timeline.

I see the `Pattern` class has a `name` field, and when I set this field the timeline display seems to register this, but -

- sometimes the names aren't shown at all

- when they are shown, it seems that it's only ever the first character that is shown

Is this a bug or am I doing something wrong ?

Best :)

(PS keep up the good work on RV, am having a lot of fun with it recently, particularly with the sampler, thinking about releasing some RV- based tracks to the Sunvox community over the medium term)

```
from rv.api import Project as RVProject
from rv.pattern import Pattern as RVPattern

from random import Random

import os

def random_color(q, offset=64, contrast=64, n=16):
    def random_color(q, offset):
        return [int(offset+q.random()*(255-offset))
                for i in range(3)]
    for i in range(n):
        color=random_color(q, offset)
        if (max(color)-min(color)) > contrast:
            return color
    return [127 for i in range(3)]

if __name__=="__main__":
    proj=RVProject()
    q=Random()
    proj.patterns=[RVPattern(lines=64,
                             tracks=1,
                             x=i*64,
                             y_size=64,
                             fg_color=(255, 255, 255),
                             bg_color=random_color(q),
                             name="%i" % (i+1))
                   for i in range(16)]
    if not os.path.exists("tmp"):
        os.mkdir("tmp")
    with open("tmp/name-demo.sunvox", 'wb') as f:
        proj.write_to(f)
```

**Comment by jhw** [comment]
@gldnspud ?

**Comment by matthewryanscott** [comment]
Resolved via #107
